docs/modules/ROOT/pages/how-tos/deleting.adoc

```diff
@@ -1 +1,41 @@
-= Deleting applications and components
\ No newline at end of file
+= Deleting applications and components
+
+You can delete an application if you are the owner or if you have admin access to the application.
+
+WARNING: If you delete an application permanently, you delete all the components and resources from the user interface (UI) and upstream repositories. You cannot undo the deletion.
+
+== Deleting an application in the web UI
+.*Prerequisites*
+
+* You have successfully signed into {ProductName}.
+* You have at least one application.
```

Comment:
These feel like no-op prerequisites. A better one might just be having the `admin` role.